encoding/meta.go

```diff
@@ -92,5 +92,6 @@ func getMetaService() *compile.ServiceSpec {
 }
 
 func getHealthSpec() (string, *compile.FunctionSpec) {
-	return metaService + "::" + healthMethod, getMetaService().Functions[healthMethod]
+	healthSpec := getMetaService().Functions[healthMethod]
+	return metaService + "::" + healthMethod, healthSpec
```

Comment:
There doesn't seem to be a functional change here, any reason these changes are included?

Reply:
An intermediate state had more returns here, but I actually simplifies it back, reverting, thanks.